### PR TITLE
Fixed `Mage_Uploader_Helper_File::getDataMaxSize()` when checking against different size units

### PR DESCRIPTION
**Pull request by F1Red5**
### Description (*)

Fixes `Mage_Uploader_Helper_File::getDataMaxSize()` when different units are used.

### Manual testing scenarios (*)

1. `min('1M', '1G')`returns `1G`

### CODE REVIEW

File: dev/tests/unit/Mage/Uploader/Helper/FileTest.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    }
+
+    public function testGetDataMaxSize(): void
+    {
+        $mock = $this->getMockBuilder(Mage_Uploader_Helper_File::class)
+            ->setMethods(['getPostMaxSize', 'getUploadMaxSize'])
+            ->getMock();
+
+        $mock->expects($this->once())->method('getPostMaxSize')->willReturn('1G');
+        $mock->expects($this->once())->method('getUploadMaxSize')->willReturn('1M');
```
Comment (pquerner) [inline dev/tests/unit/Mage/Uploader/Helper/FileTest.php:86]: What does this actually test here?

Comment (F1Red5) [reply dev/tests/unit/Mage/Uploader/Helper/FileTest.php:86]: It is to simulate php.ini values for `post_max_size` and `upload_max_filesize` when post size is set to 1GB and upload size to 1MB.

Comment (pquerner) [reply dev/tests/unit/Mage/Uploader/Helper/FileTest.php:86]: Thanks!

**Review by fballiano — APPROVED** [review]

**Comment by fballiano** [comment]
when porting to `next` I'll not include the polyfills since they're not needed (`next` is running on 8.2 minimum)

**Pull request merged**